lib/puppet/functions/vault_lookup/lookup.rb

```diff
@@ -1,3 +1,5 @@
+require 'puppet_x/vault_lookup/lookup'
```

Comment:
@joshcooper if I remember you correctly, types/providers/facts/report processors should always use require_relative to load other files, to ensure environment isolation?

Reply:
(If I'm correct we should really work on a custom rubocop rule to detect this)

Reply:
Thanks, I've updated this to sue `require_relative`.